agent/runner/jobs/backup_location.go

```diff
@@ -19,8 +19,8 @@ type BackupLocationType string
 
 // BackupLocation types. Same as in managed/models/location_model.go.
 const (
-	S3BackupLocationType        BackupLocationType = "s3"
-	PMMClientBackupLocationType BackupLocationType = "pmm-client"
+	S3BackupLocationType         BackupLocationType = "s3"
+	FilesystemBackupLocationType BackupLocationType = "pmm-client"
```

Comment:
```suggestion
	FilesystemBackupLocationType BackupLocationType = "filesystem"
```

Reply:
Double) Looks like I really have to change it)))

Reply:
Double) Looks like I really have to change it)))